src/DateComparison.php

```diff
@@ -4,21 +4,20 @@
 
 namespace CultuurNet\CalendarSummaryV3;
 
-use DateTime;
 use DateTimeImmutable;
 use DateTimeZone;
 
 final class DateComparison
 {
-    public static function onSameDay(DateTime $date1, DateTime  $date2): bool
+    public static function onSameDay(DateTimeImmutable $date1, DateTimeImmutable  $date2): bool
     {
         $date1 = $date1->setTimezone(new DateTimeZone(date_default_timezone_get()));
```

Comment:
This can now return a `boolean`. There seems to be a difference between `setTimezone` on `DateTime` and `DateTimeImmutable`

Reply:
I think that's a mistake in the stubs embedded in PHPStorm. The PHP docs don't mention that it can return `false` https://www.php.net/manual/en/datetimeimmutable.settimezone.php

It should be the same as `DateTime::setTimezone()`, which also doesn't return `bool`

Reply:
\+ PHPStan doesn't complain about it